minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "server_wildcards": [
+      "play.crystopia.net"
+    ],
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
+      "web_support": "https://labymod.net/wiki",
+      "twitter": "crystopianet",
+      "discord": "https://dc.crystopia.net",
```

Comment:
Custom links are reserved for partners; use https://discord.gg/[INVITE]

Reply:
I didn't know that, I'll fix it today

Reply:
I didn't know that, I'll fix it today